lib/logstash/outputs/opensearch/http_client/manticore_adapter.rb

```diff
@@ -27,6 +38,28 @@ def initialize(logger, options={})
       options[:cookies] = false
 
       @client_params = {:headers => DEFAULT_HEADERS.merge(options[:headers] || {})}
+      @port =  options[:port] || 9200
+      @protocol =  options[:protocol] || 'http'
+      @region =   options[:region] || 'us-east-1'
+
+      if options[:auth_type] != nil && options[:auth_type]["type"] == "basic_auth"
```

Comment:
What if auth_type is nil and user and password are passed like `options["user"]`? Isnt this valid?

Reply:
If auth_type is nil, then it is same as our existing configuration. Then, the user and password are not passed in options.

Reply:
I will keep AWS iam as only supported auth type for this PR and add basic auth for another PR

Reply:
Got it. Will Update the PR accordingly.